### ISSUE TITLE
rtl8723de with kernel 5.3.x

### ISSUE THREAD

**Issue by anauta** [lwfinger/rtlwifi_new#494]
I've been using this repo to build the wifi driver for my Realtek d723. The rtl8723de module has worked through various 5.0.x kernels, but it failed to install with the upgrade of the linux-hwe kernel to 5.3 that just came in Ubuntu 18.04. When I boot into kernel 5.3 I have no wifi.

In [issue #482 there's a comment](https://github.com/lwfinger/rtlwifi_new/issues/482#issuecomment-536695967) saying: _"With kernel 5.3.x, you should use the driver in the kernel, not this one. This repo is for people that use kernels too old to support new devices or features."_

So, do I have to uninstall the rtl8723de module in order for the driver in the kernel to work? I've gone back to kernel 5.0 for now, but I'm unsure about what to do ahead. To complicate things, I've found [this repo](https://github.com/smlinux/rtl8723de) which says it's "Realtek RTL8723DE module for Linux kernel version >= 5.0".

I'm afraid of doing something stupid, please help!

**Comment by Feanor1397** [comment]
there was update a month ago that fixed build for 5.3.x kernels. Try to make git pull from sources dir. or make a fresh clone. and don't forget to checkout to extended branch

**Comment by anauta** [comment]
Sorry, could you give more detailed instructions? Would it be something like this?
```bash
cd rtlwifi_new #this dir already exists
git checkout https://github.com/lwfinger/rtlwifi_new/tree/extended
make
sudo make install
sudo modprobe -r rtl8723de
sudo modprobe rtl8723de
```

**Comment by Feanor1397** [comment]
```bash
cd rtlwifi_new #this dir already exists
git checkout extended
git pull
make
sudo make install
sudo modprobe -r rtl8723de
sudo modprobe rtl8723de
```

**Comment by anauta** [comment]
Thanks a lot, problem solved! Running smoothly on kernel 5.3.0-19.
